src/constants.js

```diff
@@ -22,7 +22,7 @@ export const NO_DATA_VALUE = 'N/A';
 
 // Getting started URL
 // eslint-disable-next-line max-len
-export const GETTING_STARTED_URL = 'https://access.redhat.com/documentation/en-us/red_hat_insights/2022/html/assessing_and_monitoring_rhel_resource_optimization_with_insights_for_red_hat_enterprise_linux/proc-ros-install_ros-getting-started';
+export const GETTING_STARTED_URL = 'https://access.redhat.com/documentation/en-us/red_hat_insights/2022/html/assessing_and_monitoring_rhel_resource_optimization_with_insights_for_red_hat_enterprise_linux/proc-ansible-install_ros-install#doc-wrapper';
```

Comment:
Observation - previous URL was pointing to getting started section. Why can't we pointing to getting started section with modified URL? 
i.e https://access.redhat.com/documentation/en-us/red_hat_insights/2022/html/assessing_and_monitoring_rhel_resource_optimization_with_insights_for_red_hat_enterprise_linux/con-ros-overview_ros-overview

Reply:
good catch @kgaikwad let me confirm with Pau and get back to you!

Reply:
discussed with Pau and team and updated the URL to above @kgaikwad